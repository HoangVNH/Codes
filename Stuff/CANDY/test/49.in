10
70
VDXDVXDDVVVVVVDVDDDXXXDVVVDDDVXDXDXXVXXVVVXDXXDXXVVDDDDXDDDXVVVXXXDXDX
59
XXVXXDVDDXXDDVVXXDDVVVVDXDXDVVDVDXVDVDDVDVDVDXVDXXVDVVVVDXV
76
XVVDDVVVXDVVVXXDVVDDXXXXDXDXXVDDXVXDDXDXDDXVXXVVXXXVDVVDXDXVXXDXVDVVXDXXVDDX
53
VXDVVDXXDDVXDDVDXXXXXVDVXXXDXXXXVDXVXXDDDVXVDDDDDXVDD
93
VDXVXVDDVDVDDXDXVDVXVVVVVVVVVXXVVVXDDXDVXVVXVDVVDDXVXXDDXDVXXXXVXDXDXDXVVXVDDDDVDVDDDXXXVVXVV
56
VDXXVVVVVVDVDVXXXXXVXDXXXVDXVDDDVVXVVDVVXVVXXDXDVXDDVDXV
65
DVVDXVXVVXXDVDVXXVXVVDXVXXVXXXDVVVXDXDVVVVVVVXXDDVXVVXXXDDVVXXDVD
66
DXVDDDVDDDXXXVVXXVVVXXDXVXDDXXDXXXDVVDVDDXVXDXXDVDXDXXDXXXVXXDDXVD
62
XVVVVDDVDXDDDDDVXVXVXXDVXXXVDVXVXDVXVXXVDDXDVDVXVXDVDVDVXDVVVX
51
XDDVDDDXXDVDDDDXVVDVDXXVDVVVXDDXVXVDXDDDDXXDVVDDXXV
