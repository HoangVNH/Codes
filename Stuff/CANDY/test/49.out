69
57
75
52
92
55
64
65
61
50
